R/agglomerate.R

```diff
@@ -328,17 +337,14 @@ setMethod("agglomerateByVariable", signature = c(x = "SummarizedExperiment"),
 #' @export
 setMethod("agglomerateByVariable",
             signature = c(x = "TreeSummarizedExperiment"),
-            function(x, MARGIN, f, archetype = 1L, mergeTree = FALSE,
-                     mergeRefSeq = FALSE, ...){
+            function(x, MARGIN, f, archetype = 1L, mergeTree = FALSE, ...){
```

Comment:
archetype is also related to mergeRefSeq so it should be moved also

Reply:
I thought about this and maybe the cleanest solution is to move mergeRefSeq and archetype to hidden parameters also from agglomerateByRank(). All agglomerate* function are in same manual page, so I think it might be hard for user to find that the parameter is available also for agglomerateByVariable() if is listed as main parameter for agglomerateByRank()

-->

So move mergeRefSeq and archetype to hidden parameters and list them under "..." in documenration

Reply:
ACK.